<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMain</class>
 <widget class="QMainWindow" name="FormMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyRogueLife2</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../.designer/backup/icon.ico</normaloff>../../../.designer/backup/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>180</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>开始旅程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRecord">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>180</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>旅程日记</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMenu">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>180</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>游戏菜单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonQuit">
    <property name="geometry">
     <rect>
      <x>659</x>
      <y>420</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>退出游戏</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MyRogueLife2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEggs">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>已发现彩蛋: 0</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon.png</normaloff>icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAbout">
    <property name="geometry">
     <rect>
      <x>589</x>
      <y>420</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>关于</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
